canvas_modules/harness/test_resources/parameterDefs/slider_paramDef.json

```diff
@@ -0,0 +1,337 @@
+{
+  "titleDefinition": {
+    "title": "Slider with Min Max"
+  },
+  "current_parameters": {
+    "slider": 50,
+    "slider_double": 0.5,
+    "slider_null": null,
+    "slider_error": 50,
+    "slider_warning": 0,
+    "hide": true,
+    "slider_hidden": 0,
+    "disable": true,
+    "slider_disable": 50
+  },
+  "parameters": [
+    {
+      "id": "disable",
+      "type": "boolean"
+    },
+    {
+      "id": "slider",
+      "type": "integer"
+    },
+    {
+      "id": "slider_double",
+      "type": "double"
+    },
+    {
+      "id": "slider_empty",
+      "type": "integer"
+    },
+    {
+      "id": "slider_undefined",
+      "type": "integer"
+    },
+    {
+      "id": "slider_null",
+      "type": "integer"
+    },
+    {
+      "id": "hide",
+      "type": "boolean"
+    },
+    {
+      "id": "slider_hidden",
+      "type": "integer",
+      "required": true
+    },
+    {
+      "id": "slider_disable",
+      "type": "integer",
+      "required": true
+    },
+    {
+      "id": "slider_warning",
+      "type": "integer",
+      "required": true
+    },
+    {
+      "id": "slider_error",
+      "type": "integer",
+      "required": true,
+      "default": 20
+    }
+  ],
+  "complex_types": [
+    {
+      "id": "sliders",
+      "parameters": []
+    }
+  ],
+  "uihints": {
+    "id": "slider",
+    "icon": "images/default.svg",
+    "label": {
+      "default": "Slider Values"
+    },
+    "parameter_info": [
+      {
+        "parameter_ref": "slider",
+        "label": {
+          "default": "Slider Testing"
+        },
+        "description": {
+          "default": "Slider Testing"
+        },
+        "control": "slider",
+        "min_value": 10,
+        "max_value": 100,
+        "increment": 10,
+        "class_name": "slider-control-class"
+      },
+      {
+        "parameter_ref": "slider_double",
+        "label": {
+          "default": "Slider Double Testing"
+        },
+        "description": {
+          "default": "Slider Double Testing"
+        },
+        "control": "slider",
+        "min_value": 0.1,
+        "max_value": 1,
+        "increment": 0.1,
+        "class_name": "slider-control-class"
+      },
+      {
+        "parameter_ref": "slider_empty",
+        "label": {
+          "default": "Empty"
+        },
+        "description": {
+          "default": "slider with parameter value set to '\"\"'"
+        },
+        "control": "slider"
+      },
+      {
+        "parameter_ref": "slider_null",
+        "label": {
+          "default": "Null"
+        },
+        "description": {
+          "default": "slider with parameter value set to 'null'"
+        },
+        "control": "slider",
+        "min_value": null,
```

Comment:
These need to be removed.  They aren't valid and cause schema validation errors.